src/tutorials/0006-anatomy-of-a-cid/06.md

```diff
@@ -1,5 +1,5 @@
 ---
-    title: "Examples"
+    title: "One hash, multiple CID versions"
```

Comment:
I wouldn't rename this lesson as such because this should not be the focus of the lesson. The focus of the lesson is to give examples of CIDs.
The last note about the CIDs representing the same content is just a small detail.

Does it look like this is the focus of the lesson? Because if that's the case I think we should re-word it then.
But if you think having a "Examples" lesson is less important than this we can keep it as is.

Reply:
The things I find most interesting about this lesson are: 
- the introduction of the CID Inspector tool 
- what I learn about the differences between CIDv0 and CIDv1 and how they both refer to the same underlying hash 

If you had named this Examples of _____ instead of Examples, what would you have named it? 

I don't find "Examples" alone to be a very compelling lesson title. It could be "Putting it all together" or "CID Inspector" or something about comparing CIDv0 to CIDv1. What do you think?

Reply:
Ah I see. That makes sense. I agree with `One hash, multiple CID versions` then!